docs/PersonalApplications.rst

```diff
@@ -1263,6 +1264,33 @@ your tasks as completed.
    Click on the notification stream, you will be taken to the detailed
    task pane in the Task Management application.
 
+.. _NewsNotifications :
+
+
+**News post in a space**
+
+You will receive the **News post in a space** notification when a user posts an article in a space of which you are member.
```

Comment:
```suggestion
You will receive the **News posted in a space** notification when an user posts an article in a space you are member of.
```

Reply:
"A user" is actually correct (https://english.stackexchange.com/a/105117). Thanks @samehaidi  !